core/service/src/client/tests/threshold/key_gen_tests.rs

```diff
@@ -172,7 +174,21 @@ async fn test_insecure_threshold_sns_compression_keygen() {
 #[tokio::test(flavor = "multi_thread")]
 #[serial]
 async fn secure_threshold_keygen_test() {
-    preproc_and_keygen(4, FheParameter::Test, false, 1, false).await;
+    preproc_and_keygen(4, FheParameter::Test, false, 1, false, None, None).await;
+}
+
+#[cfg(feature = "slow_tests")]
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn secure_threshold_keygen_test_crash_online() {
+    preproc_and_keygen(4, FheParameter::Test, false, 1, false, None, Some(vec![2])).await;
```

Comment:
Could we add a decryption call afterwards to ensure that parties are indeed able to operate afterwards? (I know we don't plan on doing it now, but ideally it should work so good to validate)